### ISSUE TITLE
Add profiles for all users

### ISSUE THREAD

**Issue by elboletaire** [vocdoni/vote-frame#53]
/profile API calls now accept adding a user handler as param. Having this, we can now create user profiles:

- [x] Adapt the Profile component to receive data from /profile or /profile/{handle}, depending on the situation
- [x] Add a new user card to the profile, showing basic user info (username, handle, etc)
- [x] Link usernames to their profiles in tops and other places

**Comment by p4u** [comment]
There are two new public calls, by user and by fid. Example:

```
curl http://localhost:8888/profile/user/dwr.eth -s | jq .                                                                                        

{
  "mutedUsers": null,
  "polls": null,
  "reputation": 100,
  "reputationData": {
    "followersCount": 217764,
    "electionsCreated": 0,
    "castedVotes": 1,
    "participationAchievement": 0
  },
  "user": {
    "userID": 3,
    "electionCount": 0,
    "castedVotes": 1,
    "username": "dwr.eth",
    "displayname": "",
    "custodyAddress": "0x6b0bda3f2ffed5efc83fa8c024acff1dd45793f1",
    "addresses": [
      "0xD7029BDEa1c17493893AAfE29AAD69EF892B8ff2",
      "0x8fc5D6AFe572fefc4EC153587B63cE543f6fa2EA"
    ],
    "signers": [
      "c887f5bf385a4718eaee166481f1832198938cf33e98a82dc81a0b4b81ffe33d"
    ],
    "followers": 217764,
    "lastUpdated": "2024-04-04T15:55:43.458Z"
  }
}
```

I'd make the profile page quite more verbose, showing even the list of signers. This is for transparency.

**Comment by elboletaire** [comment]
Closed in 8ce8d1ad221d88ed426d80df87e0413f686e9d94 and 8073d335b4212e9e76d4f83d51c2a44609f30ae3
